### PR TITLE
Create 2020_Summer_RA_Summary.md

### PR DESCRIPTION
**Pull request by Zeyuan-Xiong**
This summary covers most of our discussions during the summer. I will update the H-S part later after I try the first few schedules. But I think it would efficient if you could first go through the report and we can discuss if there are something missing.

Those files mentioned in the report have been uploaded on G:/ drive

Thanks for your time!

### CODE REVIEW

File: 2020_Summer_RA_Summary.md [hunk 1]
```diff
@@ -1,31 +1,25 @@
 # Summary
 
+## Softwares involved: Adobe Pro; Rstudio; Excel (VBA)
```
Comment (kbuzard) [inline 2020_Summer_RA_Summary.md:3]: Please add the versions of each of these three piece of software

Also, software is the plural of software. I can't begin to imagine why, but that's English for you

Comment (kbuzard) [reply 2020_Summer_RA_Summary.md]: I saw that you changed "softwares" to "software," but did you add the version information?

What I'm thinking of is like what is at the top of this documentation file Yimin wrote : https://github.com/kbuzard/political-uncertainty/blob/master/REPLICATION.org

File: 2020_Summer_RA_Summary.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+* One efficient way to do the check is to search by words in specific columns in the EXCEL file by "ctrl+F", and pay attention to how the subparagraphs are organized, especially when threre is a change in the classification of the subparagraphs. 
+
+## Subparagraphs
+
+* The subparagraphs situations are documented under the column **"Extra"** too, and the mainly unsolved problem is that sometimes the paragraphs converge in the next round and sometime it diverge in the next round (Geneva -> Annecy -> Torquay). One potential solution is to extend the lines that diverge and put the uniform tax rate for all the branches.
+
+* One example could be paragraph 1301, in which I used extrapolation in this part to fulfill the blank of classified sub-paragraphs by using the tax rate of unclassified sub-paragraphs (see the repeated tax rate). To be more specific, for example Geneva data has no classification of the value (not mentioned in Geneva file), then I apply the same tax rate for "singles" for both under and above 80 cents. For now for other paragraphs, however, I remain them the same and put note in the **"Extra"** column.
```
[the remark this answers is not in the document]
Comment (kbuzard) [reply 2020_Summer_RA_Summary.md]: I think what is still needed here is an explanation of what you mean by "classified" and "unclassified" -- this is a term that could mean very many different things and that is the source of the problem.

**Comment by kbuzard** [comment]
I've gone through all the open comments on this pull request and either closed them or added any clarification if I think it would be helpful. Please go through the "Conversation" list and let me know if you have any questions about the open issues. I'd like to get all these wrapped up so I can merge this pull request

File: 2020_Summer_RA_Summary.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+* <font size=4>There are several paragraphs that are not very matched with the Geneva(Annecy) description, those files are listed in file ***"paragraphs"*** and painted with color in the ***"Schedules_1-15_units_changeds"***. Some of paragraphs are involved with a change of the unit across(within) rounds and they are all listed (they have been changed already, but if you want to trace them, they are listed in the file).</font>
+
+* <font size=4>I created a seperated folder called ***"UNTC_814"*** in the G:/drive, the 814 is the registration number of all the GATT files. The folder includes all the rounds files from the source of UN treaty collections. And follow the volume number, more related information can be found on the website. And other information related to the change of policy could be found in other volumes under the GATT project in UNTC.</font>
```
Comment (kbuzard) [inline 2020_Summer_RA_Summary.md:20]: Please add the url for the website

File: 2020_Summer_RA_Summary.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+* The column "**Extra**" captures the difference of between Torquay rounds and Annecy, Geneva. Most of the contents of the Extra are the followings:
+1. change of subparagraphs;
+2. special clarification;
+3. different description;
+4. bounds of tax rate.
+
+## Change of subparagraphs
+
+* Typical changes of the subparagraphs involve with the differentiation of percentage, value of size. For example for paragraph 370, in Geneva47 file, there is not specification of the value of the engine, but in Torquay file, there are two (three) subparagraphs setting different tax rate on engines of different value. Similar cases for other standards like size and percentage.
```
Comment (kbuzard) [inline 2020_Summer_RA_Summary.md:41]: The first sentence would be clearer (if I've understood correctly what you mean) to say "Typical changes in the subparagraphs involve differentiation by percentage, value or size"

File: 2020_Summer_RA_Summary.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+* One example could be paragraph 1301, in which I used extrapolation in this part to fulfill the blank of classified sub-paragraphs by using the tax rate of unclassified sub-paragraphs (see the repeated tax rate). To be more specific, for example Geneva data has no differentiation of the value (not mentioned in Geneva file), then I apply the same tax rate for "singles" for both under and above 80 cents. This paragraph 1301 example gives a view of what the columns will be like if we use extrapolation. However currently for other paragraphs, I leave them unchanged as the original documents and just put notes in the **"Extra"** column.
+
+* Another potential solution will include putting in dummy variables
+
+## Special clarification
+
+* Some products have special properties that needs to be specifically specified, and usually these properties remain the same across different rounds.
```
Comment (kbuzard) [inline 2020_Summer_RA_Summary.md:51]: How is this specification of special properties done?
Where would one look for it (i.e. which variables)? Is this in the Extra column?
Please give an example so I can understand better what this means.

File: 2020_Summer_RA_Summary.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+* The existing files have **NOT** covered the part of Cuba preferential trade agreements and also the IRC code products.
+
+* But it should be prominently noted, that we are not going to get a good time series on these from the first four rounds if they aren't in the first two rounds. They make some weird references to Smoot Hawley, so I think this will need to be explored at some point
+
+* Notice that in the later rounds, the IRC coded product still exist, but in few amounts.
+ 
+## File type
+
+* The default setting of the file is the TSV file, namely set TAB as default seperated values. The txt file run after the R code can then be converted into tsv file in EXCEL.
```
Comment (kbuzard) [inline 2020_Summer_RA_Summary.md:91]: The default setting of WHICH file?
This small paragraph makes me realize that there is nothing here explaining about the R code. Should we reference documentation that Ross wrote? Did you write any new code?

File: 2020_Summer_RA_Summary.md [hunk 7]
```diff
@@ -88,7 +88,7 @@ In general, those paragraphs have been manually filled in and checked, but I am
  
 ## File type
 
-* The default setting of the file is the TSV file, namely set TAB as default seperated values. The txt file run after the R code can then be converted into tsv file in EXCEL.
+* After run the R code contributed by Ross Jestrab (see codes details in Ross's report), we first manually reorganize the txt result (R code output). The default setting of the file should be the TSV file, namely to set TAB as default seperated values during the manual reorganization process. The TSV file is more convenient to process under EXCEL and Rstudio environment for further operation.
```
Comment (kbuzard) [inline 2020_Summer_RA_Summary.md:91]: What does it mean to manually reorganize? Why do you do this reorganization? Are you just changing column widths so it's easier to read, or something more?

**Comment by Zeyuan-Xiong** [comment]
So after running the R code (the digitization process), the txt output is really messy because the file itself usually is not in good quality, and also we need to reorganize the lines and seperate values (using TAB), after the manual reorganization (or like polishing), the txt file will be in a tidy order. Then we use EXCEL to read the file (reading as TSV) and change the columns width and so on in EXCEL to make it more readable and convenient to edit.

I am not very sure whether Ross took the exact steps as I described above for schedules 1-15, but I used them to develop the schdule 16 and it looks the same as Ross's file so at least this one will work.

**Comment by kbuzard** [comment]
> So after running the R code (the digitization process), the txt output is really messy because the file itself usually is not in good quality, and also we need to reorganize the lines and seperate values (using TAB), after the manual reorganization (or like polishing), the txt file will be in a tidy order. Then we use EXCEL to read the file (reading as TSV) and change the columns width and so on in EXCEL to make it more readable and convenient to edit.
> 
> I am not very sure whether Ross took the exact steps as I described above for schedules 1-15, but I used them to develop the schdule 16 and it looks the same as Ross's file so at least this one will work.

So is the right reference for the details "file:///G:/MAX-Filer/Collab/gradualism-kbuzard-S20/Admin/torquay/Coding-Torquay.html" and the section of it "Making a Master Excel File for Torquay Data"? How does this relate to the section "Manual Cleaning"?

**Comment by Zeyuan-Xiong** [comment]
Yes, what I said in this section is basically repeating the steps that Ross's described in his own report, but just in my own words. I thought it would be more comprehensive if I described the whole digitization process briefly.

**Comment by kbuzard** [comment]
I believe this is a good idea too. I'm just still confused about the relationship between the two sections of Ross's documentation "Making a Master Excel File for Torquay Data" and "Manual Cleaning". Is what you're describing here a combination of both those sections, or only one of them?

I also just realized that I need you to be more specific about "the R code." Can you fill in the name and location of the file(s)?

**Comment by Zeyuan-Xiong** [comment]
Thanks, I would say my description mostly covers both parts, but in short words. In Ross's report, the "Manual Cleaning" refers to the step that after runing the code, we actually need to first create the column names, design what columns we want for analysis, and then put all the messy information into the corresponding columns, sperate them with TAB; while the "Making a Master Excel File for Torquay Data" describes the process of after creating the columns in Notepad++ and put them in good shape, then open the file in excel and make it more editable and readable.  But just one thing I want to add is that in Ross's final step in "Making a Master Excel File for Torquay Data" that the "CSV" file may not be the only option or the best option, we can also directly read the excel file in R or use "TSV" file option. But it is just my own thought, I am not 100% sure and it is not a very important detail, so I did not write that in the formal report

**Comment by kbuzard** [comment]
Thanks! I think this is where it needs to be, so I'm going to merge this pull request.

**Pull request merged**